demos/html/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "dev": "wds -r ../.. -o ./demos/html/index.html --watch"
```

Comment:
hmm. This will only work for a single html file. But we have multiple now.

Reply:
I think in this case it's ok to have one script per example. This package.json is only for examples so it's not really a problem if it has many scripts.